### PR TITLE
HIVE-26447 (addendum): Fix orc_map_key_repeating.q.out location

### PR DESCRIPTION
**Pull request by jfsii**
### What changes were proposed in this pull request?
HIVE-26447 added a new test, but it is failing due to q.out being in the wrong place (and a dangling q.out being reported).

### CODE REVIEW

**Comment by jfsii** [comment]
The tests passed - I can't commit so I will leave it up to @sunchao (or another committer to commit).

I also verified the test was ran and passed - http://ci.hive.apache.org/job/hive-precommit/job/PR-3509/lastCompletedBuild/testReport/org.apache.hadoop.hive.cli.split16/TestMiniLlapLocalCliDriver/Testing___split_12___PostProcess___testCliDriver_orc_map_key_repeating_/

**Review by sunchao — APPROVED** [review]

**Comment by sunchao** [comment]
Merged, thanks @jfsii !

**Pull request merged**
